static/js/main.js

```diff
@@ -1,16 +1,21 @@
 ;(function () {
-  var langPickerTogglerElement = document.querySelector('.lang-picker-toggler')
-  var langPickerElement = document.querySelector('.lang-picker')
-  var langElements = langPickerElement.querySelectorAll('button')
+  const langPickerTogglerElement = document.querySelector(
```

Comment:
The changes here will break on older browsers.

Reply:
Do you mean `const`? It's even in IE11 https://caniuse.com/const

Reply:
I know, but IIRC the docs were supposed to work on as many browsers as possible.